view/frontend/templates/embed.phtml

```diff
@@ -51,11 +51,12 @@ use Nosto\Tagging\Block\Embed;
  * Please see a reference guide such as https://github.com/Nosto/nosto-magento2/wiki
  */
 
-/** @var Embed $block */
+/**
+ * @var Embed $block
+ * @var Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
+ */
 ?>
 <?php if ($block->getNostoScriptUrl()): ?>
   <!--suppress JSUnresolvedVariable -->
-  <script type="text/javascript"
-          src="<?= $block->escapeUrl($block->getNostoScriptUrl()); ?>"
-          async></script>
+    <?= $secureRenderer->renderTag('script', ['type' => 'text/javascript', 'src' => $block->escapeUrl($block->getNostoScriptUrl()), 'async'], '', false) ?>
```

Comment:
This change seems to be throwing an error when loading magento 


![image](https://github.com/user-attachments/assets/f571c49f-c534-45a7-a5f6-a1c88b136727)